lib/src/instances/option.dart

```diff
@@ -64,16 +184,40 @@ class Some<T> extends Option<T> {
   Some(this.value) : super._();
 
   @override
+  T get first => value;
+
+  @override
   int get hashCode => value.hashCode;
 
   @override
+  bool get isEmpty => false;
+
+  @override
+  bool get isNotEmpty => true;
```

Comment:
Can be implemented in terms of `isEmpty` avoiding the need for implementation in implementations